wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -332,7 +332,7 @@ private fun parseFields(
       label = label,
       type = type,
       name = field.name,
-//      defaultValue = field.defaultValue,
+      defaultValue = if (field.hasDefaultValue()) field.defaultValue else null,
```

Comment:
We probably need to verify that their existing test protos cover all the cases for this. The way that the default value is encoded in the descriptor is kind of funny -- it may differ from how Wire's `ProtoParser` represents it.

Reply:
I had another PR which does this and I was able to merge that + pull into this branch to test. The defaults seem to be these here: https://github.com/buildbreaker/wire/blob/ac/default-value-parse/wire-library/wire-tests/src/commonTest/proto/kotlin/all_types.proto#L103-L118